v2/pkg/protocols/headless/request.go

```diff
@@ -69,6 +70,12 @@ func (request *Request) executeRequestWithPayloads(inputURL string, payloads map
 	}
 	defer instance.Close()
 
+	if request.options.Options.Debug || request.options.Options.DebugResponse {
+		gologger.Debug().Msgf("Protocol request variables")
```

Comment:
same here

Reply:
Also (I haven't dig deep in the code), just want to know make sure the condition is correct/intentional. It everywhere says `DebugResponse` and `request variables`, but the `DebugRequest` is not mentioned.